gulp/tasks/lint.js

```diff
@@ -47,9 +47,17 @@ gulp.task( 'lint:scripts', () => _genericLint( configLint.src ) );
  * Lints all the js files for errors
  */
 gulp.task( 'lint',
-  gulp.parallel(
-    'lint:build',
-    'lint:tests',
-    'lint:scripts'
-  )
+  // Checks to see if a flag for a specific file is present, if so, use it
+  // Else, returns a list of sane default tasks to run
+  (() => {
+    let argv = minimist( process.argv.slice( 3 ) );
+    if(argv.l !== undefined) {
```

Comment:
You should be able to fix the TravisCI failure by changing this to:

```
let UNDEFINED;
if ( argv.l !== UNDEFINED ) {
```

For better or worse, we have https://eslint.org/docs/rules/no-undefined set.